src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -309,6 +291,12 @@ def _set_time_step_attributes(
             # Initialize default values for time steps per gyroperiod and Courant parameter
             self.setup_adaptive_time_step()
 
+        sync = (
```

Comment:
Since there are more ways the tracker can be configured now, I tried to add some more printouts to confirm the configuration at runtime.